docs/guides/update-channels.md

```
@@ -0,0 +1,71 @@
+place to be.
+
+## Which channel am I on?
+
+```sh
+quern set-channel
+```
+
+With no argument it prints the current channel and the branch it tracks, plus
+the valid names:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)